.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -71,6 +71,7 @@ If the code communicates with devices, web services, or third-party tools:
       Updated and included derived files by running: `python3 -m script.hassfest`.
 - [ ] New or updated dependencies have been added to `requirements_all.txt`.  
       Updated by running `python3 -m script.gen_requirements_all`.
+- [ ] For the updated dependencies- a link to the changelog, or at minimum a diff between library versions is added to the PR description.
```

Comment:
```suggestion
- [ ] For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description.
```

Reply:
Or maybe colon?

Reply:
Colon vs en dash is a style issue at this stage. Only other difference would be American vs British spelling. If you use American spelling as standard, we should remove the spaces around the dash and then use an em dash (longer). But as it is now, this seems ok.